ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -62,43 +62,58 @@
 
 (def external-interface-table-columns
   ;; [label width value-fn]
-  [[:table-header-external-interface "10%"
+  [[:table-header-external-interface "20%"
     (comp #(common-ui/linkify % % {:target "_blank"}) ::t-service/url ::t-service/external-interface)]
    [::t-service/format-short "10%" ::t-service/format]
    [::t-service/license "10%" ::t-service/license]
    [::t-service/external-service-description "10%"
     (comp #(t-service/localized-text-for "FI" %) ::t-service/description ::t-service/external-interface)]])
 
+(defn parse-content-value [value-array]
+  (let [data-content-value #(str (tr [:enums ::t-service/interface-data-content %]) ", ")
+        value-str (apply str (map #(data-content-value %) value-array))
+        return-value (if (< 45 (count value-str))
```

Comment:
Separate the "shorten this text to max length of N" functionality from the concrete location where the operation is to be applied. That makes the shortening part reusable and the code clearer.

Also, you may want to use `(str/join ", " values)` instead of `(apply str (map ...` approach.

Reply:
I made function to common-ui and replaced apply str with str/join